release-notes/5.0/preview/api-diff/rc1/Asp.Net/5.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Core.md

```diff
@@ -0,0 +1,22 @@
+# Microsoft.AspNetCore.Server.Kestrel.Core
+
+``` diff
+ namespace Microsoft.AspNetCore.Server.Kestrel.Core {
+     public class Http2Limits {
++        public TimeSpan KeepAlivePingDelay { get; set; }
+-        public TimeSpan KeepAlivePingInterval { get; set; }
+
+     }
+     public class KestrelServer : IDisposable, IServer {
+-        public KestrelServer(IOptions<KestrelServerOptions> options, IEnumerable<IConnectionListenerFactory> transportFactories, IEnumerable<IMultiplexedConnectionListenerFactory> multiplexedFactories, ILoggerFactory loggerFactory);
+
+     }
+     public class ListenOptions : IConnectionBuilder, IMultiplexedConnectionBuilder {
```

Comment:
Given that IMultiplexedConnectionBuilder has been removed, shouldn't it be removed from the above line?

```suggestion
     public class ListenOptions : IConnectionBuilder {
```